src/Entity/Formation/Article.php

```diff
@@ -79,6 +81,19 @@ class Article implements EntityMediaInterface
      */
     private $axe;
 
+    /**
+     * @var Collection|File[]
+     *
+     * @ORM\OneToMany(targetEntity="AppBundle\Entity\Formation\File", cascade={"persist"}, mappedBy="article", orphanRemoval=true)
```

Comment:
`orphanRemoval=true` => The file will be deleted if it is unlinked from this Article.

You also need to set a DELETE cascade => The file will be deleted is the Article is deleted.
( `cascade={"persist", "remove"}` or `cascade={"all"}` might be great here )